_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,128 @@
+
+To create the [Docker volume](https://docs.docker.com/storage/volumes/) where the cluster will store its data:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+docker volume create roach-single
+~~~
+
+
+
+### Step 2. Start the cluster
+
+1. Start the cluster node.
+
+    The following command starts a single-node cluster that:
+ 
+    - Stores its data in the `roach-single` volume on the Docker host, which is mounted on the `/cockroach/cockroach-data` directory within the container.
+    - If the `/cockroach/cockroach-data` within the container is empty, the specified database, user, and password are created automatically. Instead of specifying each value directly by using the `-e` or `--env` flag, you can store them in a file on the Docker host. Use one key-value pair per line and set the `--env-file` flag to the file's path.
+    - Bind-mounts the `~/init-scripts` directory on the Docker host onto the `/docker-entrypoint-initdb.d` directory within the container. Initialization scripts stored in this directory are run after CockroachDB starts and the default database, user, and password are initialized.
+    - Accepts database client connections on IP address 172.18.0.3 (which resolves to the hostname `roach-single` within the Docker container) on port 26257.
+    - Accepts connections to the DB Console on IP address 172.18.0.3 (which resolves to the hostname `roach-single` within the Docker container) on port 8080.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker run -d \
+              --env COCKROACH_DATABASE={DATABASE_NAME} \
+              --env COCKROACH_USER={USER_NAME} \
+              --env COCKROACH_PASSWORD={PASSWORD} \
+              --name=roach-single \
+              --hostname=roach-single \
+              --ip=172.18.0.3 \
+              --advertise-addr=172.18.0.3 \
+              -p 26257:26257 -p 8080:8080 \
+              -v "roach-single:/cockroach/cockroach-data" \
+              -v "~/init-scripts:/docker-entrypoint-initdb.d" \
+              cockroachdb/cockroach:latest start-single-node
+    ~~~
+
+    By default, a `certs` directory is created and CockroachDB starts in secure mode. To prevent this, add the `--insecure` flag after the `start-single-node` sub-command.
+
+    {{site.data.alerts.callout_info}}
+    The `COCKROACH_DATABASE`, `COCKROACH_USER`, and `COCKROACH_PASSWORD` environment variables and the contents of the `/docker-entrypoint-initdb.d` directory are ignored if you use `cockroach start` rather than `cockroach start-single-node`. They are also ignored if the `/cockroach/cockroach-data` directory within the container is not empty.
+    {{site.data.alerts.end}}
+
+1. Perform a one-time initialization of the cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker exec -it roach-single ./cockroach init --insecure
```

Comment:
I don't think with `start-single-node`, we will need this step. `cockroach init` is usually used for multi-node cluster.